### ISSUE TITLE
A11y_AzureDataStudioSQL_PublishProjects_Generate Scripts(Terminal)_Keyboard: Horizontal scrollbar is not scrollable using keyboard when screen is zoomed to 200%

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15098]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.1.1;#AINotInScope;#Zoom;

### Prerequisite: 

- Set the text size to 200% in "Ease of access settings"

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab an press shit+f10 on any created project.
5. Tab till "Publish Project" and hit enter
6. Hit on Generate script and navigate to task
7. Check if scrollbar is getting scrolled using keyboard

### Actual:
Horizontal scrollbar is not scrollable using keyboard when screen is zoomed to 200%

### Expected:
Horizontal scrollbar should bet scrollable using keyboard when screen is zoomed to 200%

### User Impact:
Keyboard only users will face difficulty in getting the information

**Comment by Shivamd1** [comment]
Issue logged in regression

**Comment by kisantia** [comment]
vscode does not support keyboard horizontal scrolling for trees https://github.com/microsoft/vscode/issues/58198, which is what the task history uses to display tasks. This problem is also present for the servers tree and file explorer trees when horizontal list scrolling is enabled in settings.

**Comment by kisantia** [comment]
opened issue in vscode https://github.com/microsoft/vscode/issues/121132

**Comment by hannahqin** [comment]
We have filed a Reflow exception request and are waiting on official approval from CELA. We received initial support from Guy Barker. I'll keep this thread updated once we hear back on the official decision.
https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844/

**Comment by alanrenmsft** [comment]
closing the issue, Azure Data Studio requested exception for WCAG 2.1: https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844/

**Comment by fsteffi** [comment]
githubtags:#closed;

reopening new bug as still reproduces 
https://github.com/microsoft/azuredatastudio/issues/22423
